akka-testkit/src/main/java/akka/testkit/JavaTestKit.java

```diff
@@ -558,7 +558,7 @@ public ReceiveWhile(Class<T> clazz, Duration max, int messages) {
       this(clazz, max, Duration.Inf(), messages);
     }
 
-    @SuppressWarnings("unchecked")
+    @SuppressWarnings("all")
```

Comment:
This was a weird one, BTW: without it there was an “unchecked warning” warning, and with it there was a “extraneous warning” warning. Luckily there is this big hammer.